src/test/scala/mesosphere/marathon/integration/MesosAppIntegrationTest.scala

```diff
@@ -111,6 +97,23 @@ class MesosAppIntegrationTest extends AkkaIntegrationTest with EmbeddedMarathonT
       waitForDeployment(deleteResult)
     }
 
+    "deploy a simple persistent pod" taggedAs WhenEnvSet(envVarRunMesosTests, default = "true") in {
+      Given("a pod with a single task and a volume")
+      val containerPath = "pst1"
+      val pod = residentPod(
+        id = "simple-persistent-pod",
+        mountPath = containerPath,
+        cmd = s"""echo "data" > $containerPath/data && while test -e foo; do sleep 5; done""")
```

Comment:
We should `test -e $containerPath/data`. This test should fail but it does not. The pod is constantly restarted.